connector/exceptionsconnector/connector_logs.go

```diff
@@ -0,0 +1,132 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exceptionsconnector
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	conventions "go.opentelemetry.io/collector/semconv/v1.6.1"
+	"go.uber.org/zap"
+)
+
+type logsConnector struct {
+	logsConsumer consumer.Logs
+	component.StartFunc
+	component.ShutdownFunc
+
+	logger *zap.Logger
+	ld     plog.Logs
+}
+
+func newLogsConnector(logger *zap.Logger) (*logsConnector, error) {
+	logger.Info("Building logs exceptionsconnector")
+
+	return &logsConnector{
+		logger: logger,
+		ld:     plog.NewLogs(),
+	}, nil
+}
+
+// Capabilities implements the consumer interface.
+func (c *logsConnector) Capabilities() consumer.Capabilities {
+	return consumer.Capabilities{MutatesData: false}
+}
+
+// ConsumeTraces implements the consumer.Traces interface.
+// It aggregates the trace data to generate logs.
+func (c *logsConnector) ConsumeTraces(ctx context.Context, traces ptrace.Traces) error {
+	c.logger.Debug("Consume traces")
+	for i := 0; i < traces.ResourceSpans().Len(); i++ {
+		rspans := traces.ResourceSpans().At(i)
+		resourceAttr := rspans.Resource().Attributes()
```

Comment:
Similarly, are you sure we should aggregate logs across resources, rather than preserving the resource attributes and our understanding of where the spans came from?

Reply:
I'm not very tuned into the use case but I could imagine wanting to retain more fine grained resolution. We can always make it configurable later though, so I don't object to this implementation.